Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/eval.py", line 9, in <module>
    from gpt import priority_v2 as priority
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 18
    base[boost_mask] + = priority_v2.boost
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/SLOC.py", line 7, in <module>
    from gpt import priority_v2 as priority
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 18
    base[boost_mask] + = priority_v2.boost
                       ^
SyntaxError: invalid syntax
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/bpp_online/cyclomatic_complexity.py", line 7, in <module>
    from gpt import priority_v2 as priority
  File "/home/dokhanhnam1199/QD/problems/bpp_online/gpt.py", line 18
    base[boost_mask] + = priority_v2.boost
                       ^
SyntaxError: invalid syntax
